app-backend/tile/src/main/resources/application.conf

```diff
@@ -12,6 +12,17 @@ tile-server {
   cache.enabled = ${?TILESERVER_CACHE_ENABLED}
 }
 
+akka {
+  blocking-scene-routes-dispatcher {
```

Comment:
If [this](http://doc.akka.io/docs/akka/snapshot/scala/general/configuration.html) is accurate, it seems as though  there is a `default-blocking-io-dispatcher` in the default `akka-actor` configuration:

```javascript
default-blocking-io-dispatcher {
      type = "Dispatcher"
      executor = "thread-pool-executor"
      throughput = 1

      thread-pool-executor {
        fixed-pool-size = 16
      }
}
```

Does it make sense to reuse that for this subset of operations vs. creating another thread pool?